http-app-dependencies/pom.xml

```diff
@@ -4,24 +4,43 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.springframework.cloud.stream.app</groupId>
 	<artifactId>http-app-dependencies</artifactId>
-	<version>2.0.2.BUILD-SNAPSHOT</version>
+	<version>2.0.3.BUILD-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<name>http-app-dependencies</name>
 	<description>Spring Cloud Stream HTTP App Dependencies</description>
 
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
-		<version>2.0.2.RELEASE</version>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
 		<relativePath/>
 	</parent>
 
+	<properties>
+		<spring-cloud-function.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-function.version>
+	</properties>
+
 	<dependencyManagement>
 		<dependencies>
 			<dependency>
 				<groupId>org.springframework.cloud.stream.app</groupId>
 				<artifactId>spring-cloud-starter-stream-source-http</artifactId>
-				<version>2.0.2.BUILD-SNAPSHOT</version>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>app-starters-function-common</artifactId>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-function-core</artifactId>
+				<version>${spring-cloud-function.version}</version>
```

Comment:
That's something wrong. I though SCSt 2.1 already depends on SC-function. Therefore we don't need these dependencies here.

Any explanations, please?

Reply:
That is correct, SCF should be brought in by SCSt